src/channel.spec.ts

```diff
@@ -218,4 +218,4 @@ describe('Test the channel bridge from iFrame to admin', () => {
 
     expect(callback).toHaveBeenCalledTimes(0)
   });
-});
\ No newline at end of file
+});
```

Comment:
Can you add also integration tests for the Entity and EntityCollection Serializer?

Reply:
![image](https://user-images.githubusercontent.com/39218577/158179157-8f94dfcf-7a6f-4f3c-af6d-edccbb49301f.png)

Reply:
Sure thing will add em 👍